doc/releases/changelog-dev.md

```diff
@@ -96,27 +95,39 @@
 
   ```
 
-<h4>Make use of more methods to map from molecules 🗺️</h4>
-
+<h4>More methods to map molecules to qubits 🗺️</h4>
 
-* Added new function `qml.bravyi_kitaev` to map fermionic Hamiltonians to qubit Hamiltonians.
+* A new function called `qml.fermi.bravyi_kitaev` has been added to map fermionic 
```

Comment:
```suggestion
* A new function called `qml.bravyi_kitaev` has been added to map fermionic 
```
Prefer to link to the top-level function (it's also the one in the [docs](https://xanaduai-pennylane--5575.com.readthedocs.build/en/5575/code/api/pennylane.bravyi_kitaev.html))